forge-gui/res/cardsfolder/upcoming/ogre_battlecaster.txt

```diff
@@ -0,0 +1,16 @@
+Name:Ogre Battlecaster
+ManaCost:2 R
+Types:Creature Ogre Shaman
+PT:3/3
+K:First Strike
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigPlay | TriggerDescription$ Whenever CARDNAME attacks, you may cast target instant or sorcery card from your graveyard by paying {R}{R} in addition to its other costs. If that spell would be put into a graveyard, exile it instead.
+SVar:TrigPlay:Mode$ Continuous | Affected$ Instant.YouCtrl,Sorcery.YouCtrl | MayPlay$ True | MayPlayAltManaCost$ R R RegularCost | EffectZone$ Battlefield | AffectedZone$ Graveyard | RememberObjects$ Targeted | SubAbility$ DTCast
+SVar:DTCast:Mode$ SpellCast | ValidCard$ Card.IsRemembered | Execute$ TrigPump | TriggerDescription$ When you cast that spell, CARDNAME gets +X/+0 until end of turn, where X is that spell's mana value.
```

Comment:
This needs to be T:Mode$ Spellcast

Reply:
Aha. I see that one now, thanks.

Reply:
No, the trigger (as well as the replacement) are not separate abilities
Rather they are part of the resolving ability (which exists independently on the stack!)
You can probably just use DelayedTrigger instead of Effect because Play already supports `ReplaceGraveyard$`